src/check_cve/check.py

```diff
@@ -88,8 +88,8 @@ def main():
             ),
             "--timeout": And(
                 Use(int),
-                lambda n: n > 1,
-                error="--timeout must be a an integer greater than 1.",
+                lambda n: n > 0,
+                error="--timeout must be a an integer greater than 0.",
```

Comment:
You have a bonus `a ` in your error message:
_--timeout must be ~~a~~ an integer greater than 0._

Reply:
Got it. 9bc0ece
thx